src/components/shared/formItems/ProjectPayButton.tsx

```diff
@@ -11,10 +11,14 @@ export default function ProjectPayButton({
     <Form.Item
       name={name}
       label={hideLabel ? undefined : 'Pay button'}
-      extra={`Text displayed on your project's "pay" button. Leave this blank to use the default.`}
```

Comment:
**suggestion**: From a UX perspective, I think we should prefer to keep help text like this as `extra` text and not in placeholders.

IMO Placeholders should exclusively be an example of the input value, not a description. What do you think?

Reply:
agree